circle.yml

```diff
@@ -9,12 +9,11 @@ dependencies:
   override:
     # Used by container_spec
     - docker pull alpine
-    - docker build -t=$PRIVATE_REGISTRY/$CIRCLE_PROJECT_REPONAME:b$CIRCLE_BUILD_NUM .
-    - bundle install
+    - make image
 
 test:
   override:
-    - bundle exec rake
+    - make test
```

Comment:
Or we could remove the image pre command and keep `make image` in dependencies. I personally like seeing the two actions in separate sections on CircleCI:

![screen-shot-2015-12-10-14-12-07-rsqemsyjit](https://cloud.githubusercontent.com/assets/123595/11725528/31c9444c-9f48-11e5-924f-2af9d3712e25.png)